timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -49,21 +51,40 @@ public long slowLockLogTriggerMillis() {
         return 10000;
     }
 
-    @JsonProperty("auth-clients")
+    /**
+     * The namespace and token credentials the server should use to authenticate basic service clients
+     */
+    @JsonProperty("service-auth-clients")
+    @Value.Default
+    public List<TimelockServiceAuthClient> serviceAuthClients() {
+        return ImmutableList.of();
+    }
+
+    /**
+     * The tokens the server should use to authenticate admin clients
+     */
+    @JsonProperty("admin-auth-clients")
     @Value.Default
-    public List<TimelockAuthClient> authClients() {
+    public List<String> adminAuthSecrets() {
         return ImmutableList.of();
     }
 
-    @JsonProperty("admin-token")
+    /**
+     * The token used to authenticate requests between internal nodes of the cluster if it's the case
```

Comment:
if what's the case?

Reply:
if it's the case timelock is running on a cluster with more than 1 node, I was thinking, I can remove the `if it's the case` part.